### ISSUE TITLE
Publish 2014-09-09-landis-rainey-royal.md

### ISSUE THREAD

**Issue by ftpavlik** [midnightbreakfast/web#164]
@nevanscott Should be ready for preview/proofing, save the cover, which I forwarded to you. Thanks!

**Comment by nevanscott** [comment]
This should be up in the next 15 minutes or so on the preview site...

**Comment by ftpavlik** [comment]
@nevanscott Looks good, but I have a few slight changes I'd like to make to my letter. I can't seem to edit it though... Is there something you have to do?

**Comment by ftpavlik** [comment]
@nevanscott Made my changes. Can you please push again?

**Comment by nevanscott** [comment]
Should be up now. What was wrong before with editing? (I’m assuming you figured it out.)

**Comment by ftpavlik** [comment]
No idea what was wrong with editing. I'm still getting used to navigating GitHub, so it could be that I was just coming at the file from the wrong path (is there a reason Small Plates live in both "Smalls" and "Magazine"?). Either way I had no problem when I tried again this morning. Preview looks good. I just made one last minor edit to the Editor's Note. Otherwise I think we're good to publish tomorrow!
